src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -223,7 +230,14 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
 
     const artwork = this.props.order.lineItems?.edges?.[0]?.node?.artwork
     const isInquiryCheckout = !artwork?.isPriceRange && !artwork?.price
-    const isEdition = artwork?.isEdition
+    const hasPrice =
+      (artwork?.listPrice?.__typename === "Money" &&
+        artwork?.listPrice?.major) ||
+      (artwork?.listPrice?.__typename === "PriceRange" &&
+        artwork?.listPrice?.maxPrice?.major &&
+        artwork?.listPrice?.minPrice?.major)
+    const showPriceOptions =
+      (artwork?.editionSets?.length ?? 0) < 2 && !!hasPrice
```

Comment:
Here's the meat and potatoes of this PR.
We are first checking if the artwork has a listed price and combining that with a check for whether the artwork has edition sets and if just one.
We'll be using this to decide whether to render the price options component or not.

Reply:
I would at least put `artwork?.listPrice?` in a variable, or even make a minPrice and maxPrice one to cut down on the repetitiveness and make it more readable.